addon/-private/data-view/radar/dynamic-radar.js

```diff
@@ -60,15 +59,15 @@ export default class DynamicRadar extends Radar {
   _updateConstants() {
     super._updateConstants();
 
-    if (this._calculatedEstimateHeight < this._minHeight) {
```

Comment:
seems odd for _minHeight to still be here in a generalized version

Reply:
Still a WIP, will get to this soon.